Python/Python.sublime-syntax

```diff
@@ -1018,6 +1018,11 @@ contexts:
     - include: illegal-assignment-expression
     - match: ':'
       scope: punctuation.separator.annotation.variable.python
+      push:
+        - meta_scope: meta.variable.annotation.python
+        - match: '(?==|$|#)'
```

Comment:
We should probably include `,` and `;` to make sure to pop annotations in lists or multiple inline expressions

```suggestion
        - match: $|(?=[,;=#])
```

```
var1: Type1, var2: Type2;print("foo")
```

Reply:
The `,` is a syntax error here.  

![image](https://user-images.githubusercontent.com/8558/197387822-0396f627-17b0-421a-9f02-d93da8ab3cfd.png)

The `;` is allowed though and should pop of course.

Reply:
Done, still green